_docs/principles/accessibility.md

```diff
@@ -3,29 +3,69 @@ title: Accessibility
 
-Rackspace products must be made accessible for everyone&mdash;regardless of a person’s abilities. We can assume from the statistics that many of our users have some form of visual impairment. [Colour Blind Awareness](http://www.colourblindawareness.org/) estimates that:
+Designing for accessibility is a best practice when using Helix because your designs will be usable by everyone regardless of whether they have any impairments.
+
+The purpose of this document is to articulate guidelines that you can use when you design control panel page. The list of guidelines is not exhaustive. Refer to [Web Content Accessibility Guidelines 2.1 (WCAG)](https://www.w3.org/TR/WCAG21/) for a complete list of guidelines.
+
+### Why accessibility is important
+
+Accessibility is more than just good design. It's about the empathy for and inclusion of all people. Consider sight. [Colour Blind Awareness](http://www.colourblindawareness.org/) estimates that:
 
 - 4.5% of the global population experience color blindness (1 in 12 men and 1 in 200 women)
 - 4% suffer from low vision (1 in 30 people)
 - 0.6% are blind (1 in 188 people)
 
-By using the following best practices, you can help ensure that your design is accessible for the visually impaired.
+With a global population of just over 7.5 billion people, 9.1% of the world's population equates to over one-half billion visually-impaired users that cannot use an inaccessible product. Inaccessible products can have a negative impact on a large percent of users.
+
+Not only can inaccessible design reflect poorly on our brand, it has the potential for legal ramifications. In 2018, there were at least 2258 lawsuits filed in federal court under Title III of the Americans with Disabilities Act (ADA), an increase of 177% of such lawsuits filed in 2017.
+
+### How Helix supports accessibility
+
+The core Helix team has taken the following steps to help ensure that your UIs are accessible:
+
+- To increase the probability that a screen reader interprets a UI correctly, we have developed components that include the appropriate `aria-*` and `role` attributes, where necessary, and use semantic markup as much as possible. These efforts alone do not guarantee screen reader accessibility, as accessibility is mostly dependent on the UI being developed.
+- To the greatest extent possible, we follow the guidelines set forth by the [WAI-ARIA Authoring Practices 1.1](https://www.w3.org/TR/wai-aria-practices/) to ensure keyboard accessibility.
+- Accessibility is built into the Helix color palette and hex codes. If you adhere to the color palette, you will be in compliance.
+- We have designed the spacing system, focus states, and typography to conform to accessibility standards.
+- All design specifications we have written support accessibility.
```

Comment:
If you think we should take out the 'We have designed the spacing system...' bullet and the 'all design specifications...' bullet, that leaves just one bullet's worth of stuff that we've done to support accessibility. Do we want to do that, or should I add 'conform to WCAG 2.1 accessibility standards' to the line 49 bullet and change 'All' to 'Most' in the line 50 bullet? @CITguy .

Reply:
What if we rephrased this bullet ('All Design...') to...

```
- We strive to design for the best accessibility in all of our design specifications.
```

We can't say for certain that our design specifications will support accessibility 100%, so this would communicate that accessibility is a "north star" that we aim for.

Reply:
We try, indeed! Ok, done.